pytab/core.py

```diff
@@ -255,6 +255,13 @@ def output_json(data, file_path):
     default=False,
     help="Enable additional debug output",
 )
+@click.option(
+    "--gpu",
+    "gpu_id",
+    type=int,
+    required=False,
+    help="Select which gpu to use for testing",
+)
 def cli(
```

Comment:
You did not parse the new cli option "gpu_id" into the main def.
This causes the Program to crash, as it gets an unexpected input.
Also the Variable "gpu_id" will not be available with that implementation.